CodeSniffer/Standards/Generic/Tests/Formatting/DisallowLongArraySyntaxUnitTest.pass.inc

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
No need to create separate fixture files for pass and fail cases. You can just have one file.